packages/zowe-explorer/CHANGELOG.md

```diff
@@ -2,18 +2,23 @@
 
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
-## `1.18.1`
-
-- Improved integration tests for greater efficiency [#1430](https://github.com/zowe/vscode-extension-for-zowe/pull/1430). Thanks @katelynienaber
-- Added a reminder to the PR template to create a PR from `master` to `next` after PR is merged into `master` branch [#1446](https://github.com/zowe/vscode-extension-for-zowe/pull/1446). Thanks @JillieBeanSim
-- Added @zowe/cli dependency to API [#1445](https://github.com/zowe/vscode-extension-for-zowe/pull/1445). Thanks @phaumer
-- Fix for the issue that caused mismatching of GitHub VSIX Zowe Explorer releases [#1442](https://github.com/zowe/vscode-extension-for-zowe/pull/1442). Thanks @katelynienaber
+## `1.19.0`
+
+- Added Imperative logger access for extenders [#1433](https://github.com/zowe/vscode-extension-for-zowe/pull/1433). Thanks @katelynienaber
+- Added documentation for Imperative logger for extenders [#1467](https://github.com/zowe/vscode-extension-for-zowe/pull/1467). Thanks @katelynienaber
+- Added support for the validate profile and credential features [#1443](https://github.com/zowe/vscode-extension-for-zowe/pull/1443). Thanks @tiantn
+- Added the profile validation and credentials check functionalities to the Zowe Explorer FTP extension [#1284](https://github.com/zowe/vscode-extension-for-zowe/pull/1284). Thanks @lauren-li
```

Comment:
This is the related issue for the 1443 PR above. I think we can remove this item